README.md

```diff
@@ -80,36 +80,59 @@ Community discussion about Triton DataCenter happens in two main places:
 
 ### Cloud on a Laptop (CoaL)
 
-An easy way to try Triton DataCenter is by downloading and installing a Cloud on a Laptop
-(CoaL) build. CoaL is a VMware virtual appliance that provides a full Triton head node for
-development and testing.
+An easy way to try Triton DataCenter is by downloading and installing a Cloud
+on a Laptop (CoaL) build. CoaL is a VMware virtual appliance that provides a
+full Triton head node for development and testing.
 
-The [CoaL Setup document](./docs/developer-guide/coal-setup.md) contains detailed instructions for
-downloading and installing the virtual appliance.
+The [CoaL Setup document](./docs/developer-guide/coal-setup.md) contains
+detailed instructions for downloading and installing the virtual appliance.
+If you use the ISO installer (see below) on a Virtual Machine, the CoaL setup
+instructions post-booting should be followed as well.
 
-If you already have a CoaL and would like to update the installation, follow the instructions
-for [updating a Triton standup using `sdcadm`](https://github.com/joyent/sdcadm/blob/master/docs/update.md).
+If you already have a CoaL and would like to update the installation, follow
+the instructions for [updating a Triton standup using
+`sdcadm`](https://github.com/joyent/sdcadm/blob/master/docs/update.md).
+
+### ISO Installer
+
+The [ISO installer page](./docs/developer-guide/iso-installer.md) contains
+information on the ISO installer. With SmartOS now able to boot off a ZFS
+pool, the Triton Head Node can do the same.
+
+### iPXE installer
+
+Some deployment environments install new hardware using an iPXE boot. An
+[iPXE boot](./docs/developer-guide/ipxe-installer.md) directory for a Triton
+Head Node can install Triton using an iPXE installation.
 
 ### Installing Triton on a Physical Server
 
-A Triton DataCenter server runs SmartOS which is a live image. This means that
-it boots from a USB flash drive (key).
-a physical USB key, inserting the key and booting the server from that key.
-To install Triton, first obtain the latest release USB build.
+A Triton DataCenter server runs SmartOS which is a live image. That live
+image, which generates a ramdisk root with many portions set to read-only,
+can boot from a USB flash drive (a physical USB key), or a ZFS pool.  To
+install a USB-key Triton, first obtain the latest release USB build.
+Otherwise, given the cautions in the [ISO installer
+page](./docs/developer-guide/iso-installer.md), a physical server can boot
+off of its `zones` pool.
+
+Both USB-key-boot and ZFS-pool-boot use the same interfaces within Triton,
+with a ZFS bootable filesystem substituting for the USB-key.
 
 
 #### Hardware
 
 For Triton development only, the minimum server hardware is:
 
 - 8 GB USB flash drive
-- Intel Processors with VT-x and EPT support (all Xeon since Nehalem)
+- Intel Processors with VT-x and EPT support (all Xeon since Nehalem), or AMD Processors no earlier than EPYC or Zen. 
```

Comment:
Is that true for AMD? I assume this is for if one want's to test bhyve.  `SVM` came in before Zen but I don't know what the minimum requirement for bhyve actually is.

Reply:
I figured I'd rather be conservative here, and yes BHYVE crossed my mind when I settled on this.